<?xml version="1.0"?>
<launch>

  <arg name="command_input_name" default="command"/>
  <arg name="fixed_frame" default="robot_map"/>

  <arg name="base_commands_file_path" default="$(find robot_simple_command_manager)/config/base_commands.yaml"/>
  <arg name="commands_file_path" default="$(find robot_simple_command_manager)/config/commands.yaml"/>

  <node name="command_manager" pkg="robot_simple_command_manager" type="simple_command_manager_node.py" output="screen">
    <param name="command_input_name" value="$(arg command_input_name)"/>
    <rosparam command="load" file="$(arg base_commands_file_path)" subst_value="true"/>
    <rosparam command="load" file="$(arg commands_file_path)" subst_value="true"/>
  </node>


</launch>
